physionet-django/console/views.py

```diff
@@ -985,7 +1074,7 @@ def gcp_bucket_management(request, project, user):
             LOGGER.info("The bucket {0} already exists, skipping bucket and \
                 group creation".format(bucket_name))
         else:
-            utility.create_bucket(project.slug, project.version, project.title, is_private)
+            create_s3_bucket(project.slug, project.version, project.title, is_private)
```

Comment:
this doesn't look right! an aws function is being called inside `gcp_bucket_management`

Reply:
@tompollard you are right! I mistakenly renamed 'utility.create_bucket' to 'create_s3_bucket'. Thanks for catching that!